### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by chfast**
Resolves https://github.com/ethereum/evmjit/issues/111 and https://github.com/ethereum/evmjit/issues/65.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,31 +7,15 @@ It can be used to substitute classic interpreter-like EVM Virtual Machine in Eth
...
-  
-### Windows
-
-Ask me.
+The EVMJIT project uses **CMake** tool to configure the build and depends only on LLVM library.
+LLVM installation is not needed, because CMake will download and build LLVM from source.
+However, LLVM requires **Python** interpreter to be built.
+
+```sh
+mkdir build; cd build
```
Comment (axic) [inline README.md]: Why not have all of them individual lines then?

File: README.md [hunk 2]
```diff
@@ -7,31 +7,15 @@ It can be used to substitute classic interpreter-like EVM Virtual Machine in Eth
...
-2. Build library with cmake
-  1. `mkdir build && cd $_`
-  2. `cmake -DLLVM_DIR=/usr/local/lib/llvm-3.7/share/llvm/cmake .. && make`
-3. Install library
-  1. `make install` (with admin rights?)
-  
-### Windows
-
-Ask me.
+The EVMJIT project uses **CMake** tool to configure the build and depends only on LLVM library.
```
Comment (axic) [inline README.md]: "on the LLVM"

File: README.md [hunk 3]
```diff
@@ -7,31 +7,15 @@ It can be used to substitute classic interpreter-like EVM Virtual Machine in Eth
...
+LLVM installation is not needed, because CMake will download and build LLVM from source.
+However, LLVM requires **Python** interpreter to be built.
+
+```sh
+mkdir build; cd build
+cmake ..
+cmake --build . --config RelWithDebInfo
```
Comment (axic) [inline README.md]: Is the `--build .` really needed?

Comment (chfast) [reply README.md]: Yes. This replaces `make`, but works also for Visual Studio.

**Pull request merged**